app/modules/parsing/knowledge_graph/qdrant_indexing_service.py

```
@@ -0,0 +1,723 @@
+    """
+    global _colbert_model, _colbert_tokenizer
+    if _colbert_model is None:
+        try:
+            from pylate.colbert import ColbertModel  # type: ignore
+
+            logger.info(f"Loading ColBERT model via pylate: {COLBERT_MODEL}")
+            _colbert_model, _colbert_tokenizer = ColbertModel.from_pretrained(
+                COLBERT_MODEL
+            )
+            logger.info("ColBERT model loaded via pylate")
+        except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)